src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -159,6 +159,8 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             // Initialize our font with the renderer
             // We don't have to care about DPI. We'll get a change message immediately if it's not 96
             // and react accordingly.
+            dxEngine->SetFontFeatures(_fontFeatures);
+            dxEngine->SetFontAxes(_fontAxes);

```

Comment:
I don't think these two are required.

Reply:
Oh I thought `UpdateSettings` would be called anyways...

Reply:
Oh I guess `UpdateSettings` is called before `Initialize`? This explains why you have to put `_fontFeatures` into a member variable instead of just making it a local variable in `UpdateSettings`...
The lifetimes of objects in this class are a bit confusing for me...

Reply:
Yeah unfortunately :/